lib/plugins/aws/invokeLocal/resolvers/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const aws = require('aws-sdk');
+
+let cfResources = null;
+
+function getCFResources(serviceConfig) {
+  if (!cfResources) {
+    const cf = new aws.CloudFormation(serviceConfig);
+    const serverless = module.exports.serverless;
+
+    cfResources = cf
+      .describeStackResources({
+        StackName: `${serverless.service.service}-${serverless.service.provider.stage}`,
+      })
+      .promise()
+      .then(resources => {
+        return resources.StackResources;
+      });
+  }
+  return cfResources;
+}
+
+function resolveResourceId(plugin, resource, resourceName, parameters, serviceConfig) {
+  return getCFResources(serviceConfig).then(cf => {
+    const cfResource = cf.find(x => x.LogicalResourceId === resourceName);
+    if (!cfResource) {
+      return parameters;
+    }
+    return cfResource.PhysicalResourceId;
+  });
+}
+
+module.exports.getCFResources = getCFResources;
+module.exports.resolveResourceId = resolveResourceId;
```

Comment:
Let's remove this file, and make this functions in this file a methods of a plugin